democracy/factories/hearing.py

```diff
@@ -84,9 +84,12 @@ class Meta:
         choices=SectionType.objects.exclude(identifier="main")
     )
     commenting = factory.fuzzy.FuzzyChoice(choices=Commenting)
+    hearing = factory.SubFactory(HearingFactory)
 
     @factory.post_generation
     def post(self, create, extracted, **kwargs):
+        if extracted:
+            return
```

Comment:
Same thing here as with `BaseCommentFactory`, except you don't have to deal with a base class. Separate this into two different factories, `SectionFactory` (same as the current one, but without the post-generation hook) and `SectionWithCommentsFactory` (subclass of `SectionFactory` with the post-generation hook from the original version)

Reply:
I think adding new simpler factories would perhaps be better way to operate this.

Reply:
for now I thought just get ridoff the `BaseCommentFactory`'s random creation time functionality and add proper names for the post hooks and give possibility to exclude them.

Reply:
Yeah, I get it... this is just something that won't ever get done. I'll make a ticket out of this, maybe the legendary someone™ will eventually finish the job. Most likely not.